src/test/java/uk/gov/companieshouse/api/strikeoffobjections/authorization/AttachmentDownloadAuthorizationInterceptorTest.java

```diff
@@ -0,0 +1,75 @@
+package uk.gov.companieshouse.api.strikeoffobjections.authorization;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import uk.gov.companieshouse.api.strikeoffobjections.common.ApiLogger;
+
+@RunWith(MockitoJUnitRunner.class)
+public class AttachmentDownloadAuthorizationInterceptorTest {
+    
+    @InjectMocks
+    private AttachmentDownloadAuthorizationInterceptor interceptor;
+
+    @Mock
+    private HttpServletRequest request;
+
+    @Mock
+    private HttpServletResponse response;
+
+    @Mock
+    private ApiLogger logger;
+
+    @Test
+    public void willAuthoriseUserToDownloadAttachmentWhenOnlyDownloadRolePresent() {
+        when(request.getHeader("ERIC-Authorised-Roles"))
+            .thenReturn("permission /admin/strike-off-objections-download");
```

Comment:
make sure these aren't the real role names (for security)

Reply:
leave as is, according to https://www.gov.uk/government/publications/open-source-guidance/when-code-should-be-open-or-closed it is ok

Reply:
Looks like this will be ok now.